src/app/storage-manager.service.ts

```diff
@@ -0,0 +1,26 @@
+import { Injectable } from '@angular/core';
+import { Todo } from './interfaces/todo';
+
+@Injectable()
+
+export class StorageManagerService {
+
+  getLocalStorage(key) {
```

Comment:
still referring to `localstorage` here and the setter,
should be something like `getItem` and `setItem`